### PR TITLE
Admin Show and Delete Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Admin Show Delete Material

### CODE REVIEW

File: resources/views/materials/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        <a href="{{ route('materials.index') }}" class="pull-right"><span
+                    class="fa fa-arrow-left btn btn-primary"></span></a>
+    </h1>
+@endsection
+@section('main-content')
+    <div class="container bootstrap snippet">
+        <div class="panel-body inf-content">
+            <div class="row">
+                <div class="col-md-4">
+                    <img class="img-circle img-thumbnail" src="{{ $material->image }}" data-original-title="Usuario">
```
Comment (ghost) [inline resources/views/materials/show.blade.php]: `Usuario` what is mean?

Comment (vhhieu95) [reply resources/views/materials/show.blade.php]: I design follow https://bootdey.com/snippets/view/Table-user-information

**Comment by vhhieu95** [comment]
I updated , please check again. Thank you so much

**Comment by ghost** [comment]
LGTM 😄

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**